src/modules/twap/utils/getPartOrderStatus.ts

```diff
@@ -0,0 +1,23 @@
+import { EnrichedOrder } from '@cowprotocol/cow-sdk'
+
+import { OrderStatus } from 'legacy/state/orders/actions'
+import { isOrderExpired, isOrderFulfilled } from 'legacy/state/orders/utils'
+
+import { TwapOrderItem, TwapOrderStatus } from '../types'
+
+export function getPartOrderStatus(
```

Comment:
I think I already asked this and the answer is no, but anyway: can part orders be partially filled?
Will we support it to be in the future?

Reply:
Nop, parts are not partially fillable:
https://github.com/rndlabs/composable-cow/blob/main/src/types/twap/libraries/TWAPOrder.sol#L84